corehq/apps/data_dictionary/util.py

```diff
@@ -264,6 +266,14 @@ def save_case_property(name, case_type, domain=None, data_type=None,
         return gettext('Unable to save valid values longer than {} characters').format(max_len)
 
 
+def delete_case_property(name, case_type, domain):
+    try:
+        prop = CaseProperty.objects.get(name=name, case_type__name=case_type, case_type__domain=domain)
+    except CaseProperty.DoesNotExist:
+        return gettext('Case property does not exist.')
```

Comment:
nit: I see we have done it elsewhere. Should translations be done in the util module or should they be limited to views?
I guess I would go for latter but don't have a strong reason against it keeping it in util, specially if its already being used.

Reply:
I think this should be fine to keep in the util module. There are other functions in this module, such as `save_case_property()`, which also use `gettext`.